_includes/whois-search.html

```diff
@@ -0,0 +1,212 @@
+<script>
+    function renderRdapContacts(entities = [], searchResultsId="rdap-search-results") {
+        const cardBody = document.createElement('div');
+        cardBody.classList.add("usa-card__body");
+
+        // maps RDAP contact field to readable display label
+        function getContactDisplayLabel(contactDetail) {
+            const contactFieldToLabel = {
+                "version": "Version",
+                "fn": "Name",
+                "org": "Organization",
+                "adr": "Address",
+                "tel_voice": "Phone",
+                "tel_fax": "Fax"
+            }
+ 
+            let contactField = contactDetail[0];
+
+            if (contactField === "tel") {
+                const phoneType = contactDetail[1]["type"][0];
+                contactField= `${contactDetail[0]}_${phoneType}`;
+            }
+
+            return contactFieldToLabel[contactField];
+        }
+
+        function displayContactData(contactDetail) {
+            let contactField = contactDetail[0]
+            let contactData = contactDetail[3]
+
+            if (contactField === "adr") {
+                if (contactData[0] === "REDACTED FOR PRIVACY") {
+                    return "REDACTED FOR PRIVACY";
+                } else {
+                    return contactData.join(" ");
+                }
+            }
```

Comment:
Are there ever going to be instances where this information isn't redacted? Every single test I tried had all this redacted. That's not really a problem, I'm just wondering.

Reply:
I think it's likely most of the data here will be redacted due to the nature of PII - I was also curious how helpful this data would be since I also found the primarily non-redacted data were the different dates of registration/expiry

Reply:
I think it's likely most of the contact data here will be redacted due to the nature of PII - I was also curious how helpful this data would be since I also found the primarily non-redacted data were the different dates of registration/expiry

Reply:
@h-m-f-t do you happen to know if it's likely if this WHOIS contact information will not be redacted? I can do some digging to see if this is redacted by default on Cloudflare's end or if it's getting redacted for another reason

Reply:
We do redact this data by default, except for `Security Email:`. While it's possible we will display some of this information in the future (most likely some of the `Registrant...` info, we don't currently have plans to do so.